state-chain/runtime/src/monitoring_apis.rs

```diff
@@ -27,7 +27,7 @@ pub struct BtcUtxos {
 
 #[derive(Serialize, Deserialize, Encode, Decode, Eq, PartialEq, TypeInfo, Debug)]
 pub struct EpochState {
-	pub blocks_per_epoch: u32,
+	pub epoch_duration: u32,
```

Comment:
Also a breaking change here, fyi @marcellorigotti

Reply:
Does this mean that after the next binary release until we run the runtime-upgrade this rpc won't be available?
Or will it still work right away but with the new field name (blocks_per_epoch -> epoch_duration)?

Reply:
Yeah it would mean that the binary release needs to be coordinated with the runtime upgrade. I think a better option would be to add a new field with the new name, and remove it in the following release (1.7 or 1.8).

Reply:
I agree with the other option. If this RPC doesn't work then monitoring doesn't work as well (the majority of it at least)